dask/array/fft.py

```diff
@@ -24,21 +24,24 @@
     """
 
 
-def _fft_wrap(fft_func, dtype, out_chunk_fn):
-    def func(a, n=None, axis=-1):
-        if len(a.chunks[axis]) != 1:
-            raise ValueError(chunk_error % (axis, a.chunks[axis]))
+def _fft_wrap(dtype, out_chunk_fn):
+    def _fft_wrapper(fft_func):
+        def func(a, n=None, axis=-1):
+            if len(a.chunks[axis]) != 1:
+                raise ValueError(chunk_error % (axis, a.chunks[axis]))
 
-        chunks = out_chunk_fn(a, n, axis)
+            chunks = out_chunk_fn(a, n, axis)
 
-        return map_blocks(partial(fft_func, n=n, axis=axis), a, dtype=dtype,
-                          chunks=chunks)
+            return map_blocks(partial(fft_func, n=n, axis=axis), a, dtype=dtype,
+                              chunks=chunks)
```

Comment:
This line was already here. I have merely indented it. While there may be improvements that can be made to this function, I'd like to keep this change compact.